src/test/java/com/oltpbenchmark/benchmarks/tpch/generation/TestGenerators.java

```diff
@@ -0,0 +1,95 @@
+import java.util.List;
+
+import junit.framework.TestCase;
+
+import com.oltpbenchmark.benchmarks.tpch.generation.*;
+
+public class TestGenerators extends TestCase {
+
+    public TestGenerators() {
+        super();
+    }
+
+    public void testGenerators() {
```

Comment:
Are you just printing the output of the generators? How are we going to check whether the generators contain correct data when the test runs automatically in Github?

Also if the default scalefactor is one, does this mean that we're going to generate 1GB of debug output?

Reply:
For automatic check I should use assertEqual with some ground truth data.

In those output I hardcoded those SFs to be very small. SFs are parameters for generators.

Reply:
There is already testcase infrastructure for automatically testing benchmark data loaders. Make a new class `TestTPCHLoader` that extends `AbstractTestLoader`:

Examples:
* https://github.com/cmu-db/benchbase/blob/main/src/test/java/com/oltpbenchmark/benchmarks/tpcc/TestTPCCLoader.java
* https://github.com/cmu-db/benchbase/blob/main/src/test/java/com/oltpbenchmark/benchmarks/tatp/TestTATPLoader.java
* thttps://github.com/cmu-db/benchbase/blob/main/src/test/java/com/oltpbenchmark/benchmarks/tpcc/TestTPCCLoader.java